cv_basics/FaceDetection[] detections
